src/main/java/bio/terra/cli/utils/Version.java

```diff
@@ -5,6 +5,10 @@ public class Version {
   /** Getter for the Terra CLI version of the current JAR. */
   public static String getVersion() {
     // read from the JAR Manifest file
-    return Version.class.getPackage().getSpecificationVersion();
+    String version = Version.class.getPackage().getSpecificationVersion();
+    if (version != null) {
+      return version;
+    }
+    return System.getProperty("TERRA_VERSION");
```

Comment:
Similar to Config.java, add a comment and a warning log https://github.com/DataBiosphere/terra-cli/blob/main/src/main/java/bio/terra/cli/businessobject/Config.java#L60